File: backend_refactoring/CSharpApi/Models/BluePrints/Beings/BlackSmith.cs
namespace CSharpApi.Models.BluePrints.Beings
{
    public class BlackSmith : Worker
    {
        public BlackSmith(string workerName, bool employed = true, string profession = "worker", bool alive = true, string fieldOfWork = "blacksmith", bool working = false, int production = 500) : base(workerName, employed, profession, alive, fieldOfWork, working, production)
        {
        }
    }
}

File: frontend/static/script/models/population/Blacksmith.js
"use strict";
import { Worker } from "./Worker.js";



export class Blacksmith extends Worker {
    constructor(population_id, name, profession, alive, employed, field_of_work, working, production) {
        super(population_id, name, profession, alive, employed, field_of_work, working, production);
        
    }
}
